Project/MyGameLibrary/Character/Character.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections.Generic;
+
+namespace Fall2020_CSC403_Project.code
+{
+    public class Character
+    {
+        public string CharacterName;
+        public int LoveScore;
+        public Dictionary<ItemID, LoveValue> ItemScores;
```

Comment:
ItemID and LoveValue should be types (Items, int), and you can comment to specify what they are